hazelcast/src/main/java/com/hazelcast/internal/management/request/MapConfigRequest.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.management.request;
+
+import com.hazelcast.config.MapConfig;
+import com.hazelcast.cluster.Member;
+import com.hazelcast.internal.management.ManagementCenterService;
+import com.hazelcast.internal.management.dto.MapConfigDTO;
+import com.hazelcast.internal.management.operation.GetMapConfigOperation;
+import com.hazelcast.internal.management.operation.UpdateMapConfigOperation;
+import com.hazelcast.internal.json.JsonObject;
+
+import java.util.Set;
+
+import static com.hazelcast.internal.management.ManagementCenterService.resolveFuture;
+import static com.hazelcast.internal.util.JsonUtil.getBoolean;
+import static com.hazelcast.internal.util.JsonUtil.getObject;
+import static com.hazelcast.internal.util.JsonUtil.getString;
+
+/**
+ * Request for updating map configuration from Management Center.
+ */
+public class MapConfigRequest implements ConsoleRequest {
```

Comment:
can be named as `UpdateMapConfigRequest` to align naming with other map-config-update operations.

Reply:
this is also used for getting the map config, so leaving as is now.